package.json

```diff
@@ -54,11 +54,14 @@
   "lint-staged": {
     "*.js": [
       "prettier --config _dev/.prettierrc --write",
-      "eslint -c _dev/.eslintrc"
+      "eslint"
```

Comment:
Correct - eslint looks recursively upwards for an .eslintrc file from the given file to be linted.

Reply:
Would need to be moved up 1 directory so that all packages technically inherit from this base directory.

Reply:
Discussed sync with Julian; he's going to move the file up a directory, so that packages that don't have any .eslintrc files of their own still get linted according to this config.